src/main/java/org/profit/telegrambot/service/CategoryService.java

```diff
@@ -0,0 +1,92 @@
+package org.profit.telegrambot.service;
+
+import org.profit.telegrambot.database.Database;
+import org.profit.telegrambot.model.Category;
+
+import java.sql.*;
+
+public class CategoryService {
+    public static Category getCategoryById(Integer id) {
+
+        loadCategoryList();
+
+        for (Category category : Database.categoryList) {
+            if (category.getId().equals(id)) {
+                return category;
+            }
+        }
+        return null;
```

Comment:
There is a cool class in JDK. It's `java.util.Optional`. That guy help us: (1) avoid NPE (stands for `NullPointerException`) and (2) increase readability.

Can we use it here (`Optional.empty()`) instead of returning `null` directly?

PS: yes, of course `return category;` will transform into `return Optional.of(category);`